cabal-install/cabal-install.cabal

```diff
@@ -239,6 +245,11 @@ executable cabal
         zlib       >= 0.5.3    && < 0.7,
         hackage-security >= 0.5 && < 0.6
 
+    if flag(old-bytestring)
+      build-depends: bytestring <  0.10.2, bytestring-builder
+    else
+      build-depends: bytestring >= 0.10.2
```

Comment:
No upper bound on bytestring-builder you mean.

@hvr this is your patch, what's it supposed to be? :-)